go.mod

```
@@ -1,30 +1,35 @@
 module github.com/docker/buildkit-syft-scanner
 
-go 1.20
+go 1.21.0
```

I don't think that's correct and should be `<major>.<minor>` no?

```suggestion
go 1.21
```

---

:thinking: odd. I set it to `go 1.21`, and `go mod tidy` produced this...